### PR TITLE
Azure support

### LINKED ISSUE #8: Add "azurerm" provider
Add the "azurerm" provider to the known providers.

### PR DESCRIPTION
**Pull request by kerak19**
Added support for Azure:
 - icons
 - logic (for connections between VNets)
 - test
 - example `tfstate`
 - example `tf`

Closes #8

### CODE REVIEW

**Comment by kerak19** [comment]
Done.

I'm sorry about rebase, but I've done something wrong and had to.

**Comment by kerak19** [comment]
Added

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 - Capability to have 2 Nodes connected with 2 edges of different directions
   ([PR #38](https://github.com/cycloidio/inframap/pull/38))
+- Azure support
+  ([PR #40](https://github.com/cycloidio/inframap/pull/40))
```
Comment (xescugc) [inline CHANGELOG.md:8]: https://github.com/cycloidio/inframap/issues/8

Comment (kerak19) [reply CHANGELOG.md]: Ok, modified the PR link to issue link

**Comment by kerak19** [comment]
Done

**Comment by kerak19** [comment]
Done. Will have to modify the commit message during rebase

**Comment by kerak19** [comment]
Done

File: provider/azurerm/azurerm.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		if !ok {
+			break
+		}
+		ins = append(ins, vnID)
+
+		rvni := cfg["remote_virtual_network_id"]
+		outs = append(outs, rvni.(string))
+	}
+	return ins, outs
+}
```
Comment (xescugc) [inline provider/azurerm/azurerm.go:68]: We have test for ths on the other providers

**Comment by kerak19** [comment]
Done

**Comment by kerak19** [comment]
Done

**Pull request merged**